Allow to omit non ready backends (#70)
IBM/varnish-operator: docs/varnish-cluster-configuration.md

|-------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|
| `affinity                                                 ` | [Affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) settings for the pods. It allows you to configure onto which nodes Varnish pods should prefer being scheduled.                                                                      | `optional`  |
| `backend.namespaces                                       ` | Namespace(s) to look for backend pods. By default - namespace the VarnishCluster is deployed to.                                                                                                                                                                                              | `required`  |

> | `backend.onlyReady                                       `  | Include (`false`, by default) or exclude (`true`) backend pods from the VCL (.Backends template var). Useful when want to respect Kubernetes health checks (by default not ready pods are also included in VCL) instead of setting Varnish readiness probes.                                  | `optional`  |

Reviewer: ```suggestion
| `backend.onlyReady                                       `  | Include (`false`, by default) or exclude (`true`) backend pods from the VCL (.Backends template var). Alters `.Backends` template variable based on Kubernetes health checks (by default not ready pods are also included in VCL) instead of [Varnish health probes](https://varnish-cache.org/docs/6.6/reference/vcl-probe.html#backend-health-probes).                                  | `optional`  |
```
